src/daemon/auth/providers/openid_connect/provider.rs

```diff
@@ -31,6 +31,8 @@ use std::{
 
 use tokio::sync::{RwLock, RwLockReadGuard};
 
+use base64::engine::Engine as _;
+use base64::engine::general_purpose::URL_SAFE_NO_PAD as BASE64_ENGINE;
```

Comment:
Is it maybe worth including "NO_PAD" or something in the import name so that a casual reader of the code who doesn't check exactly what is imported doesn't miss that this `BASE64_ENGINE` is not the same as the `BASE64_ENGINE` used in other files?

Reply:
Good idea!